912_Sort_an_Array.py

```diff
@@ -0,0 +1,36 @@
+#O(n) T(nlogn)
+from typing import List
+
+
+class Solution:
+    def sortArray(self, nums: List[int]) -> List[int]:
```

Comment:
There are many algorithms to sort array. Investigate some of them and implement them.